randovania/game_description/game_migration.py

```diff
@@ -312,6 +312,18 @@ def _migrate_v28(data: dict, game: RandovaniaGame) -> None:
                     node["kind"] = hint_types[node["kind"]]
 
 
+def _migrate_v29(data: dict, game: RandovaniaGame) -> None:
+    for region in data["regions"]:
+        for area in region["areas"].values():
+            for node in area["nodes"].values():
+                if node["node_type"] == "pickup":
+                    custom_index_group = None
+                    if game.value == "prime2":
```

Comment:
would you mind adding a comment pointing out that cave story is specifically being excluded? (I would like it to be but it might seem like a mistake without acknowledging it)

Reply:
Is the in_dark_aether thing still actually being used in CS and not just a forgotten extra?

Reply:
only being used to create virtual regions. it's a relic of the time when connections between regions were much more limited. no reason not to convert them to real regions now

Reply:
but I would like it to be excluded from this specific migration since I would actually prefer if the new split regions are grouped separately by the generator